config_samples/lido_dao_holesky_config.json

```diff
@@ -1,115 +1,175 @@
-        "@aragon/apps-vault": {
-            "url": "https://github.com/lidofinance/aragon-apps/",
-            "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
-            "relative_root": "apps/vault"
-        },
-        "@aragon/apps-agent": {
-            "url": "https://github.com/lidofinance/aragon-apps/",
-            "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
-            "relative_root": "apps/agent"
-        },
-        "openzeppelin-solidity": {
-            "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
-            "commit": "06e265b38d3e9daeaa7b33f9035c700d6bc0c6a0",
-            "relative_root": ""
-        },
-        "solidity-bytes-utils": {
-            "url": "https://github.com/GNSPS/solidity-bytes-utils",
-            "commit": "9776282d181839fbb4b18f2cf218e316d6df871c",
-            "relative_root": ""
-        },
-        "@openzeppelin/contracts": {
-            "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
-            "commit": "fa64a1ced0b70ab89073d5d0b6e01b0778f7e7d6",
-            "relative_root": "contracts"
-        },
-        "@openzeppelin/contracts-v4.4": {
-            "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
-            "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
-            "relative_root": "contracts"
-        }
+    "@aragon/apps-lido": {
+      "url": "https://github.com/lidofinance/aragon-apps/",
+      "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
+      "relative_root": ""
+    },
+    "@aragon/apps-finance": {
+      "url": "https://github.com/lidofinance/aragon-apps/",
+      "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
+      "relative_root": "apps/finance"
+    },
+    "@aragon/apps-vault": {
+      "url": "https://github.com/lidofinance/aragon-apps/",
+      "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
+      "relative_root": "apps/vault"
+    },
+    "@aragon/apps-agent": {
+      "url": "https://github.com/lidofinance/aragon-apps/",
+      "commit": "b09834d29c0db211ddd50f50905cbeff257fc8e0",
+      "relative_root": "apps/agent"
+    },
+    "openzeppelin-solidity": {
+      "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
+      "commit": "06e265b38d3e9daeaa7b33f9035c700d6bc0c6a0",
+      "relative_root": ""
+    },
+    "solidity-bytes-utils": {
+      "url": "https://github.com/GNSPS/solidity-bytes-utils",
+      "commit": "9776282d181839fbb4b18f2cf218e316d6df871c",
+      "relative_root": ""
+    },
+    "@openzeppelin/contracts": {
+      "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
+      "commit": "fa64a1ced0b70ab89073d5d0b6e01b0778f7e7d6",
+      "relative_root": "contracts"
+    },
+    "@openzeppelin/contracts-v4.4": {
+      "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
+      "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
+      "relative_root": "contracts"
+    }
+  },
+  "raise_exception": true,
```

Comment:
I find this name being too vague tbh (as said above)

Reply:
raise_exceptions_on_comparison